mix.exs

```diff
@@ -25,9 +25,13 @@ defmodule MongoosePush.Mixfile do
 
   defp deps do
     [
-     {:pigeon, github: "rslota/pigeon"},
-     {:chatterbox, github: "rslota/chatterbox", override: true},
-     {:maru, "~> 0.12", override: true},
+     {:pigeon, github: "rslota/pigeon", ref: "f85b74e"},
+     {:chatterbox, github: "rslota/chatterbox", ref: "4a8abc9", override: true},
+
+     {:maru,  github: "rslota/maru", ref: "54fc038", override: true},
+     {:cowboy,  "~> 2.3", override: true},
+     {:jason, "~> 1.0"}, # This should be included by cowboy, but it's not for some reason
```

Comment:
You can remove the comment here 🙂  It'll be included in the next release of Phoenix, but cowboy has nothing to do with it 🤠

Reply:
I don't use Phoenix - how is that relevant? And you're right, it's not cowboy's fault, but I'm not sure which lib is at fault here. Cowboy is the one that crashes because of lack of this lib.